test/functional/device/motion-e2e-specs.js

```diff
@@ -1,61 +1,61 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
+import { PREDICATE_SEARCH } from '../helpers/element';
 import { MOCHA_TIMEOUT, initSession, deleteSession } from '../helpers/session';
-import { SETTINGS_CAPS } from '../desired';
-
-
-const PREDICATE_SEARCH = '-ios predicate string';
+import { SETTINGS_CAPS, amendCapabilities } from '../desired';
 
 chai.should();
 chai.use(chaiAsPromised);
 
-describe('ReduceMotion', function () {
+describe('ReduceMotion', function() {
   this.timeout(MOCHA_TIMEOUT);
 
-  let caps = Object.assign({usePrebuiltWDA: true}, SETTINGS_CAPS);
+  let caps = amendCapabilities(SETTINGS_CAPS, { 'appium:usePrebuiltWDA': true });
```

Comment:
same as above. assign `caps` in beforeEach to avoid unexpected side effects

Reply:
fixed https://github.com/appium/appium-xcuitest-driver/commit/239f4eee9420f092ef34f0482614f7cacd17d242